jbpm-db-scripts/src/main/resources/db/ddl-scripts/derby/derby-jbpm-schema.sql

```diff
@@ -803,6 +805,7 @@
     create index IDX_NInstLog_pInstId on NodeInstanceLog(processInstanceId);
     create index IDX_NInstLog_nodeType on NodeInstanceLog(nodeType);
     create index IDX_NInstLog_pId on NodeInstanceLog(processId);
+    create index IDX_NInstLog_pId on NodeInstanceLog(workItemId);
```

Comment:
typo, it should be IDX_NInstLog_workItemId

Reply:
ok